src/cogs/pin.py

```diff
@@ -16,10 +16,10 @@ class Pin(Cog):
     async def pin(interaction: Interaction, message: Message):
         if message.pinned():
             await message.unpin()
-            await interaction.response.send_message("Message désépinglé.",ephemeral=True)
+            await interaction.response.send_message(f"Message {message.jump_url} désépinglé.", ephemeral=True)
         else:
             await message.pin()
-        await interaction.response.defer()
+        	await interaction.response.defer()
```

Comment:
```suggestion
            await interaction.response.defer()
```
Mybad c'est mieux 4 spaces